src/lib/helpers/date.ts

```diff
@@ -73,6 +73,53 @@ export const toLocaleTimeISO = (datetime: string | number) => {
     return date.toLocaleTimeString('sv');
 };
 
+export const utcHourToLocaleHour = (utcTimeString: string) => {
+    const now = new Date();
+    const [hours, minutes] = utcTimeString.split(':').map(Number);
+    const customTime = Date.UTC(now.getFullYear(), now.getMonth(), now.getDate(), hours, minutes);
+
+    const options: Intl.DateTimeFormatOptions = {
+        hour: 'numeric',
+        minute: 'numeric',
+        hourCycle: 'h23'
+    };
+
+    return new Intl.DateTimeFormat('en', options).format(new Date(customTime));
+};
+
+export const utcWeekDayToLocaleWeekDay = (utcDay: string, utcTimeString: string) => {
+    const now = new Date();
+
+    // easy mapping instead of passing numbers.
+    const dayMap = {
+        Sunday: 0,
+        Monday: 1,
+        Tuesday: 2,
+        Wednesday: 3,
+        Thursday: 4,
+        Friday: 5,
+        Saturday: 6
+    };
+
+    const utcDayNumber = dayMap[utcDay];
+    const [hours, minutes] = utcTimeString.split(':').map(Number);
+
+    const daysUntilNextUtcDay = (7 + utcDayNumber - now.getUTCDay()) % 7;
+    now.setUTCDate(now.getUTCDate() + daysUntilNextUtcDay);
+    now.setUTCHours(hours, minutes, 0, 0);
+
+    return new Intl.DateTimeFormat('en', { weekday: 'short' }).format(now);
+};
+
+export const localeTimezoneShortHand = () => {
+    const options: Intl.DateTimeFormatOptions = {
+        timeZoneName: 'long'
+    };
+
+    const dateWithTimezone = new Intl.DateTimeFormat('en', options).format(new Date());
+    return dateWithTimezone.match(/\b([A-Z])/g).join('');
```

Comment:
I'm worried about the regex and how the capital letters might not match up with the actual timezone acronym. Let me discuss this with the team.

Reply:
For the timezone, let's show a [tooltip icon](https://github.com/appwrite/console/blob/afb50f720d94d0d92a4807d1931624fc55b41386/src/lib/elements/forms/label.svelte#L31-L38) and put the `longGeneric` timezone in there.

Reply:
I guess the tooltip would go for both `support` & `supportWizard`, right?

Reply:
I've added a tooltip to both, but the icon on the support page moves to the next line, kinda breaks the flow. I could alter the text or something to fit beside the timings text.

Reply:
@ItzNotABug, make sure to re-request a review whenever you need me to take a look at anything again, including a comment.